src/ocean/text/arguments/TimeIntervalArgs.d

```diff
@@ -198,9 +198,10 @@ public long parseTimeArgument ( cstring op ) ( cstring value, long reference_tim
     an iso8601 date, a timestamp, `now` or `yesterday`.
 
     If only one argument is provided, the second one will be infered:
+        - `yesterday` it's rewriten as `1d/d 1d`
```

Comment:
The commit "Implement yesterday keyword as relative time" appears to be changing some previous behaviour, rather than implementing a new behaviour. The commit message should describe what the old behaviour was and the reason for changing it.

Reply:
It sounds like this could potentially be a breaking change, but without under understanding the old behaviour and the rationale behind changing it, it's difficult to say.

If it does turn out to be a breaking change, we could introduce a new argument, in addition to `"yesterday"`.